packages/manager/src/utilities/pricing/dynamicPricing.ts

```diff
@@ -0,0 +1,86 @@
+import { getVolumePrice } from 'src/utilities/pricing/entityPricing';
+
+import { BACKUP_PRICE, LKE_HA_PRICE, NODEBALANCER_PRICE } from './constants';
+
+import type { Region } from '@linode/api-v4';
+import type { FlagSet } from 'src/featureFlags';
+
+interface CalculatePriceOptions {
+  increaseValue: number;
+  initialPrice: number;
+}
+
+export interface DataCenterPricingOptions {
+  entity: 'Backup' | 'LKE HA' | 'Nodebalancer' | 'Volume';
+  flags: FlagSet;
+  regionId: Region['id'];
+  size?: number;
+}
+
+// The key is a region id and the value is the percentage
+// increase in price.
+const priceIncreaseMap = {
+  'br-gru': 30,
+  'id-cgk': 20,
+};
```

Comment:
```suggestion
const priceIncreaseMap = {
  'br-gru': 30, // Sao Paulo
  'id-cgk': 20, // Jakarta
};
```
I think it would be helpful to comment this for our own convenience/readability.